mozci/queries/group_durations.query

```diff
@@ -0,0 +1,13 @@
+from: unittest
+groupby:
+    - task.id
+    - result.group
+select:
+    - {name: result_duration, value: result.duration, aggregate: sum}
```

Comment:
Maybe Kyle can answer this better, but this need `aggregate: sum`? Shouldn't there only be a single duration per result? I could very well be mistaken.

Reply:
The query is correct: We are adding the durations of all the tests in a group